code/game/objects/items/devices/spy_bug.dm

```diff
@@ -17,7 +17,6 @@
 	origin_tech = "{'programming':1,'engineering':1,'esoteric':3}"
 
 	var/obj/item/radio/spy/radio
-	var/obj/machinery/camera/spy/camera
```

Comment:
While you're in the area it might be worth changing the examine() text to remove the 'must be attached to device' requirement.